libc/src/locale/localeconv.cpp

```diff
@@ -0,0 +1,49 @@
+//===-- Implementation of localeconv --------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/locale/localeconv.h"
+
+#include "src/__support/CPP/limits.h"
+#include "src/__support/common.h"
+#include "src/__support/macros/config.h"
+
+namespace LIBC_NAMESPACE_DECL {
+
+static char DOT_STRING[] = ".";
+static char EMPTY_STRING[] = "";
+
+static struct lconv POSIX_LCONV = {
```

Comment:
These constants should be defined as `constexpr`.
```suggestion
static constexpr char DOT_STRING[] = ".";
static constexpr char EMPTY_STRING[] = "";

static constexpr struct lconv POSIX_LCONV = {
```

Reply:
The standard states that these be `char *`, which means that if someone wrote to them it would segfault if we made them const.